### ISSUE TITLE
Text Classification (30 marks)

### ISSUE THREAD

**Issue by jefroy** [jefroy/big-data-A4#3]
1.	When choosing a metric to access the performance of your classifier provide a brief explanation of why you chose that metric. 
2.	Perform the following classification experiments keeping track of the performance of each classification task for future use: 
i.	Logistic regression model on word count 
ii.	Logistic regression model on TFIDF 
iii.	Logistic regression model on TFIDF + ngram 
iv.	Support Vector Machine model on word count 
v.	Support Vector Machine model on TFIDF 
vi.	Support Vector Machine model on TFIDF + ngram 
You may use the [SVM ](http://scikit-learn.org/stable/modules/generated/sklearn.svm.LinearSVC.html#sklearn.svm.LinearSVC)classifier from sklearn 
3.	Plot a bar graph showing the performance of each of the experiments.
